src/libraries/Microsoft.Extensions.Configuration.Binder/src/README.md

```diff
@@ -0,0 +1,91 @@
+# Microsoft.Extensions.Configuration.Binder
+
+Provides the functionality to bind an object to data in configuration providers for [Microsoft.Extensions.Configuration](https://www.nuget.org/packages/Microsoft.Extensions.Configuration/). This package enables you to represent the configuration data as strongly-typed classes defined in the application code. To bind a configuration, use the [Microsoft.Extensions.Configuration.ConfigurationBinder.Get](https://learn.microsoft.com/dotnet/api/microsoft.extensions.configuration.configurationbinder.get) extension method on the `IConfiguration` object. To use this package, you also need to install a package for the [configuration provider](https://learn.microsoft.com/dotnet/core/extensions/configuration#configuration-providers), for example, [Microsoft.Extensions.Configuration.Json](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.Json/) for the JSON provider.
+
+Documentation can be found at https://learn.microsoft.com/dotnet/core/extensions/configuration
+
+## Contribution Bar
+- [x] [We consider new features, new APIs, bug fixes, and performance changes](../../../libraries/README.md#primary-bar)
+
+The APIs and functionality are mature, but do get extended occasionally.
+
+## Deployment
+[Microsoft.Extensions.Configuration.Binder](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.Binder/) is not included in the shared framework. The package is deployed as out-of-band (OOB) and needs to be installed into projects directly.
```

Comment:
>  is not included in the shared framework

Could you please change this to be ` is not included in the ASP.NET core shared framework`.

We'll need to fix this in the rest of readmes.

Reply:
I assume you meant "is included..."? It's actually there looking at shipped binaries. But it is not included in regular Netcoreapp shared framework. So shouldn't it be something like this:

> [Microsoft.Extensions.Configuration.Binder](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.Binder/) is included in ASP.NET Core shared framework, but not in regular .NET shared framework. The package is deployed as out-of-band (OOB) and needs to be installed into projects directly when not using ASP.NET Core.

Reply:
Updated, didn't noticed the latest review suggestion when commenting.

Reply:
I confirmed this library is part of ASP.NET Core Runtime